ambari-server/src/main/java/org/apache/ambari/server/audit/event/LoginAuditEvent.java

```diff
@@ -62,15 +62,14 @@ private LoginAuditEventBuilder() {
     protected void buildAuditMessage(StringBuilder builder) {
       super.buildAuditMessage(builder);
 
-      builder.append(", Operation(User login), Roles(").append(System.lineSeparator());
+      builder.append(", Operation(User login), Roles(");
 
       if (roles != null && !roles.isEmpty()) {
         List<String> lines = new LinkedList<>();
         for (Map.Entry<String, List<String>> entry : roles.entrySet()) {
           lines.add("    " + entry.getKey() + ": " + StringUtils.join(entry.getValue(), ", "));
         }
-        builder.append(StringUtils.join(lines, System.lineSeparator()));
-        builder.append(System.lineSeparator());
+        builder.append(StringUtils.join(lines, " , "));
```

Comment:
This may make for a confusing list of values...
```
entry1: value1a, value1b,   entry2: entry2a...  
```

Maybe use a `;` instead...
```
entry1: value1a, value1b;   entry2: entry2a...  
```